python/cuml/manifold/t_sne.pyx

```diff
@@ -302,10 +299,19 @@ class TSNE(Base):
         self.pre_learning_rate = learning_rate
         self.post_learning_rate = learning_rate * 2
 
-        self._should_downcast = should_downcast
         return
 
-    def fit(self, X):
+    def __getattr__(self, attr):
+        # Function only to check attribute Y
+        # while will be deprecated in 0.14
+        if attr == "Y":
```

Comment:
for a single attribute, maybe it would be clearer to have a property?

```
@property
def Y(self):
 # warn then return self.embedding
```

getattr can lead to _extremely_ confusing issues at times

Reply:
That said, it's great to see the fix to use `embedding_` instead!